meson.build

```diff
@@ -12,7 +12,7 @@ cc = meson.get_compiler('c')
 
 add_project_arguments('-D_POSIX_C_SOURCE=200809L', language: 'c')
 
-jsonc = dependency('json-c', version: '>=0.13', fallback: ['json-c', 'json_c'])
+jsonc = dependency('json-c', version: '>=0.14', fallback: ['json-c', 'json_c'])
```

Comment:
I think it's fine to bump the version. Debian stable has 0.12.1 anyways, so the subproject fallback is already required on these systems.